models/data_warehouse.model.lkml

```diff
@@ -392,15 +392,27 @@ explore: calls_events {
   view_label: "Calls Events Telemetry "
   view_name: calls_events
 
-  join: dates {
-    sql_on: ${calls_events.timestamp_date} = ${dates.date_date} ;;
-    relationship: many_to_one
+  join: license_server_fact {
+    from: license_server_fact
+    view_label: "License Server Fact"
+    sql_on: ${license_server_fact.server_id} = ${calls_events.user_id};;
+    relationship: one_to_many
+    fields: [license_server_fact.customer_name_unlinked, license_server_fact.edition, license_server_fact.max_edition, license_server_fact.license_email]
   }
 
-  join: license_server_fact {
-    sql_on: ${calls_events.user_id} = ${license_server_fact.server_id} ;;
+  join: excludable_servers {
+    from: excludable_servers
+    view_label: "Server Fact"
+    sql_on: ${excludable_servers.server_id} = ${calls_events.user_id};;
+    relationship: one_to_many
+    fields: [excludable_servers.reason]
+  }
+
+  join: dates {
+    from: dates
+    view_label: " Dates"
```

Comment:
Do we want to specify what this `Dates` is for?

Reply:
The dates table is raw, its used for join conditions and to report on various date requirements, for example - Day of the week. Its also used to make complex calculations easy by providing all the dates of the Calendar.

Reply:
Right, but will this label appear as `Dates` in Looker?

Based on the query, it appears to be specific for Calls

```sql
    sql_on: ${calls_events.timestamp_date}::date <= ${dates.date_date}::date AND ${calls_events.timestamp_date}::date >= ${dates.date_date}::date - INTERVAL '30 DAYS' AND ${dates.date_date}::DATE <= CURRENT_DATE::DATE ;;
```